gossip/simulation/network.go

```diff
@@ -99,12 +99,6 @@ func NewNetwork(nodeCount int, networkType string,
 		gossipNode.Start(leftNode.Server, stopper)
 		stopper.AddCloser(leftNode.Server)
 
-		// Node 0 gossips gossip count.
```

Comment:
You should probably change the network simulator to gossip the node id with just a placeholder value since we're now counting the number of gossiped node IDs--doesn't matter for tests unless you have more than 1000 nodes, as that min will apply, but would be more correct.